repos/fdbt-netex-output/package.json

```diff
@@ -12,7 +12,7 @@
     },
     "husky": {
         "hooks": {
-            "pre-commit": "NODE_ENV=production lint-staged && npm run test:ci"
+            "pre-commit": ". ${NVM_DIR}/nvm.sh && nvm use && NODE_ENV=production lint-staged && npm run test:ci"
```

Comment:
How come this was required?

Reply:
Whenever I was running tests but had switched to node 14 (used by the site etc), the netex-output tests were failing. This ensures the correct node version is being used when running those tests.